packages/gatsby-theme-docs/src/templates/release-notes-list.js

```diff
@@ -40,12 +52,20 @@ const ReleaseNotesListTemplate = (props) => (
                   </Markdown.TypographyPage>
                 </LayoutReleaseNote>
               ))}
-          </SpacingsStack>
-        </div>
-      </Markdown.TypographyPage>
-    </LayoutReleaseNotesList>
-  </ThemeProvider>
-);
+            </SpacingsStack>
+          </div>
+        </Markdown.TypographyPage>
+      </LayoutReleaseNotesList>
+    </ThemeProvider>
+  );
+
+  function scrollToTop() {
+    const layoutPage = document.querySelector('#top');
+    layoutPage.scrollIntoView({
+      block: 'start',
+    });
+  }
```

Comment:
I would define and use this function as an utility function, there is no need to pass it as a prop.